docs/release-notes/8.16.asciidoc

```diff
@@ -56,6 +56,28 @@ On November 12, 2024, it was discovered that manually running a custom query rul
 ====
 // end::known-issue[]
 
+// tag::known-issue-53[]
+[discrete]
+.Alerts page crashes if you upgrade to 8.16 and access it in a non-default space  
+[%collapsible]
+====
+*Details* +
+On November 14, 2024, it was discovered that the Alerts page would crash if you upgraded to 8.16 and accessed the page in a non-default space.
```

Comment:
I think specifying that the`Unable To Load Page` error would display or something to that effect

Reply:
Added in [5394c8d](https://github.com/elastic/security-docs/pull/6169/commits/5394c8d9acc1046ccb38113501760346bbef3ae5)